tests/model_selection/test_hyperband.py

```diff
@@ -0,0 +1,185 @@
+import numpy as np
+import scipy.stats
+from sklearn.linear_model import SGDClassifier
+
+import dask
+import dask.array as da
+from dask.distributed import Client, wait
+from distributed.utils_test import loop, cluster, gen_cluster
+from tornado import gen
+
+from dask_ml.datasets import make_classification
+from dask_ml.model_selection import HyperbandCV
+from dask_ml.wrappers import Incremental
+from dask_ml.model_selection._hyperband import _partial_fit
+from dask_ml.utils import ConstantFunction
+from distributed.metrics import time
+
+import pytest
+
+
+@pytest.mark.parametrize("array_type,library", [("dask.array", "dask-ml"),
+                                                ("numpy", "sklearn"),
+                                                ("numpy", "test")])
+def test_sklearn(array_type, library, loop, max_iter=27):
+    with cluster() as (s, [a, b]):
+        with Client(s['address'], loop=loop):
+            chunk_size = 100  # make dask array with one chunk
+            d = 20
+            X, y = make_classification(n_samples=chunk_size, n_features=d,
+                                       random_state=42, chunks=chunk_size)
+            if array_type == "numpy":
+                X = X.compute()
+                y = y.compute()
+                chunk_size = X.shape[0]
+
+            sgd_params = {'alpha': np.logspace(-2, 1, num=1000),
+                          'l1_ratio': np.linspace(0, 1, num=1000),
+                          'average': [True, False]}
+            kwargs = dict(tol=-np.inf, penalty='elasticnet', random_state=42)
+            if library == "sklearn":
+                model = SGDClassifier(**kwargs)
+                params = sgd_params
+            elif library == "dask-ml":
+                model = Incremental(SGDClassifier(**kwargs))
+                params = sgd_params
+            elif library == "test":
+                model = ConstantFunction()
+                params = {'value': np.linspace(0, 1, num=1000)}
+            else:
+                raise ValueError
+
+            search = HyperbandCV(model, params, max_iter=max_iter,
+                                 random_state=42)
+            search.fit(X, y, classes=da.unique(y))
+
+            score = search.best_estimator_.score(X, y)
```

Comment:
This should (hopefully) be identical to `search.score(X, y)`.

However, right now I get

```pytb
AttributeError                            Traceback (most recent call last)
<ipython-input-25-2ccecb1ce4ac> in <module>()
----> 1 alg.score(X, y)

~/sandbox/dask-searchcv/dask_searchcv/model_selection.py in score(self, X, y)
    793     @derived_from(BaseSearchCV)
    794     def score(self, X, y=None):
--> 795         if self.scorer_ is None:
    796             raise ValueError("No score function explicitly defined, "
    797                              "and the estimator doesn't provide one %s"

AttributeError: 'HyperbandCV' object has no attribute 'scorer_'
```

Probably need to set `scorer_` in your `fit` method.

Reply:
Could you try that, and verify that the scores match?

Reply:
Ah, I see have you a TODO about this already.